src/redux/wallet/actions.js

```diff
@@ -69,56 +70,48 @@ export const updateTIMEBalance = () => async (dispatch) => {
 }
 
 export const updateTIMEDeposit = () => async (dispatch) => {
+  dispatch({type: WALLET_TIME_DEPOSIT_FETCH})
   const dao = await contractsManagerDAO.getTIMEHolderDAO()
   const deposit = await dao.getAccountDepositBalance(ls.getAccount())
   dispatch({type: WALLET_TIME_DEPOSIT, deposit})
 }
 
-// TODO This is only for test purposes
-// export const requireTIME = () => async (dispatch) => {
-//   dispatch(hideModal())
-//   dispatch(balanceFetch(TIME))
-//   try {
-//     await TokenContractsDAO.requireTIME()
-//     const token = await contractsManagerDAO.getTIMEDAO()
-//     return dispatch(updateBalance(token))
-//   } catch (e) {
-//     dispatch(balanceFetch(TIME))
-//   }
-// }
+export const requireTIME = () => async (dispatch) => {
+  try {
+    const dao = await contractsManagerDAO.getAssetDonator()
+    await dao.requireTIME()
+  } catch (e) {
+    dispatch(showAlertModal({title: 'Require TIME error', message: e.message}))
```

Comment:
we should add animated spinner to this action, so dispatch FETCH action before try and replace this showAlertModal string on another dispatch FETCH
also add dispatch FETCH after dao.requireTIME() inside try